devices/ePaper/Graphics.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+using System;
+
+using Iot.Device.ePaper.Buffers;
+using Iot.Device.ePaper.Drivers;
+using Iot.Device.ePaper.Fonts;
+using Iot.Device.ePaper.Primitives;
+
+namespace Iot.Device.ePaper
+{
+    /// <summary>
+    /// A graphics class for ePaper displays with basic graphic APIs support.
+    /// </summary>
+    public sealed class Graphics : IDisposable
```

Comment:
Graphics seems quite familiar with the nanoFramework.Graphics nuget. did you took inspiration? It's really to understand at what point both can be "compatible". It does not need but when it's close, it's always goood